### PR TITLE
feat: add upgrade not for warning after release 0.38

### PR DESCRIPTION
**Pull request by guilhem-barthes**
Fixes FL-1606

### CODE REVIEW

**Review by SdgJlbl — APPROVED** [review]

File: docs/source/how-to/deploying-substra/upgrade-notes.rst [hunk 1]
```diff
@@ -4,6 +4,38 @@
...
+   DETAIL:  The database was created using collation version 2.31, but the operating system provides version 2.36.                                                                                                        
+   HINT:  Rebuild all objects in this database that use the default collation and run ALTER DATABASE <db_name> REFRESH COLLATION VERSION, or build PostgreSQL with the right library version.
+
+#. Gather required info:
+
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$POD_NAME``, can be obtained from ``kubectl get pods -n $NAMESPACE``)
+
+#. Run the following commands
+   #. For the backend:
```
Comment (thbcmlowk) [inline docs/source/how-to/deploying-substra/upgrade-notes.rst:30]: I think this is not intented?

image

---

```suggestion
#. Run the following commands

   #. For the backend:
```

File: docs/source/how-to/deploying-substra/upgrade-notes.rst [hunk 2]
```diff
@@ -4,6 +4,38 @@
...
+
+   .. code-block:: bash
+
+      kubectl exec -n $NAMESPACE $POD_NAME -- sh -c 'PGPASSWORD=$POSTGRES_PASSWORD psql -U $POSTGRES_USER $POSTGRES_DATABASE -c "REINDEX DATABASE;" -c " ALTER DATABASE \"$POSTGRES_DATABASE\" REFRESH COLLATION VERSION;"'
+
+   #. For the orchestrator, use the previous command replacing `POSTGRES_PASSWORD` and `POSTGRES_USER` by their value
```
Comment (thbcmlowk) [inline docs/source/how-to/deploying-substra/upgrade-notes.rst:36]:
```suggestion
   #. For the orchestrator, use the previous command replacing ``POSTGRES_PASSWORD`` and ``POSTGRES_USER`` by their value
```

File: docs/source/how-to/deploying-substra/upgrade-notes.rst [hunk 3]
```diff
@@ -4,6 +4,38 @@
...
+.. code-block::
+
+   WARNING: database "<db_name>" has a collation version mismatch                                                                                                                                                     │
+   DETAIL:  The database was created using collation version 2.31, but the operating system provides version 2.36.                                                                                                        
+   HINT:  Rebuild all objects in this database that use the default collation and run ALTER DATABASE <db_name> REFRESH COLLATION VERSION, or build PostgreSQL with the right library version.
+
+#. Gather required info:
+
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$POD_NAME``, can be obtained from ``kubectl get pods -n $NAMESPACE``)
```
Comment (thbcmlowk) [inline docs/source/how-to/deploying-substra/upgrade-notes.rst:27]: I find it a bit strange to have the note when the warning is expected 
> you may get the following warning:

---

```suggestion
This version upgrades the integrated PostgreSQL databases chart. After this uprade, you may get the following warning:

.. code-block::

   WARNING: database "<db_name>" has a collation version mismatch                                                                                                                                                     │
   DETAIL:  The database was created using collation version 2.31, but the operating system provides version 2.36.                                                                                                        
   HINT:  Rebuild all objects in this database that use the default collation and run ALTER DATABASE <db_name> REFRESH COLLATION VERSION, or build PostgreSQL with the right library version.

.. note::

   You need to apply the following command for the ``substra-backend`` and the ``orchestrator``
   
#. Gather required info:

   * Kubernetes namespace (here ``$NAMESPACE``)
   * Helm release (hereafter ``$POD_NAME``, can be obtained from ``kubectl get pods -n $NAMESPACE``)
```

**Review by thbcmlowk — CHANGES_REQUESTED** [review]
Thank for the fix! Some minor comment thought 🙏

File: docs/source/how-to/deploying-substra/upgrade-notes.rst [hunk 4]
```diff
@@ -4,6 +4,38 @@
 Upgrade notes
 *************
 
+.. _ops upgrade notes 0.38:
+
+Substra 0.38.0
+--------------
+
+This version upgrades the integrated PostgreSQL databases chart. After this uprade, you may get the following warning:
+.. code-block::
```
Comment (thbcmlowk) [inline docs/source/how-to/deploying-substra/upgrade-notes.rst:13]: image

---

```suggestion
This version upgrades the integrated PostgreSQL databases chart. After this uprade, you may get the following warning:

.. code-block::
```

File: docs/source/how-to/deploying-substra/upgrade-notes.rst [hunk 5]
```diff
@@ -4,6 +4,38 @@
...
+
+#. Run the following commands
+
+   #. For the backend:
+
+   .. code-block:: bash
+
+      kubectl exec -n $NAMESPACE $POD_NAME -- sh -c 'PGPASSWORD=$POSTGRES_PASSWORD psql -U $POSTGRES_USER $POSTGRES_DATABASE -c "REINDEX DATABASE;" -c " ALTER DATABASE \"$POSTGRES_DATABASE\" REFRESH COLLATION VERSION;"'
+
+   #. For the orchestrator, use the previous command replacing ``POSTGRES_PASSWORD`` and ``POSTGRES_USER`` by their value
```
Comment (thbcmlowk) [inline docs/source/how-to/deploying-substra/upgrade-notes.rst:36]: Increment does not seem to work but I'm not sure I know how to fix it
image

It's working [here](https://github.com/Substra/substra-documentation/blob/main/docs/source/how-to/deploying-substra/walkthrough/20-orchestrator-deployment.rst?plain=1#L70-L72) and looks the same to me 
 
---

```suggestion
   #. For the backend:

   .. code-block:: bash

      kubectl exec -n $NAMESPACE $POD_NAME -- sh -c 'PGPASSWORD=$POSTGRES_PASSWORD psql -U $POSTGRES_USER $POSTGRES_DATABASE -c "REINDEX DATABASE;" -c " ALTER DATABASE \"$POSTGRES_DATABASE\" REFRESH COLLATION VERSION;"'

   #. For the orchestrator, use the previous command replacing ``POSTGRES_PASSWORD`` and ``POSTGRES_USER`` by their value
```

**Review by thbcmlowk — APPROVED** [review]
LGTM, thanks for updating!

**Pull request merged**